src/components/editgrid/fixtures/comp15.js

```diff
@@ -0,0 +1,161 @@
+export default {
+'_id': '64901198a11ba83edcab8d72',
+'title': 'Grid',
+'name': 'grid',
+'path': 'grid',
+'type': 'form',
+'display': 'form',
+'tags': [],
+'access': [
```

Comment:
I would prefer not to have such a long form. For renderer test it is enough your form to have only components, type, display, title, path, name and id. All other properties can be deleted.